src/deploy/functions/release/reporter.ts

```diff
@@ -233,7 +233,12 @@ export function triggerTag(endpoint: backend.Endpoint): string {
     return `${prefix}.taskQueue`;
   }
 
+  if (backend.isCallableTriggered(endpoint)) {
+    return `${prefix}.callable`;
+  }
+
   if (backend.isHttpsTriggered(endpoint)) {
+    // NOTE: Legacy trigger annotation relies on a special label to differentiate http vs callable triggers.
```

Comment:
What if we instead modified triggerParser to just create a callableTriggered function so we don't have two deployment paths?

Reply:
I can see that it will be cleaner, but this requires quite a bit of plumbing here and in the Emulator side of things.

I can work on it as a follow up to the PR instead.